node/src/BaseClient.ts

```diff
@@ -909,22 +927,10 @@ export class BaseClient {
             this.promiseCallbackFunctions[callbackIndex] = [
                 (resolveAns: T) => {
                     try {
-                        if (resolveAns instanceof PointerResponse) {
-                            if (typeof resolveAns === "number") {
-                                resolveAns = valueFromSplitPointer(
-                                    0,
-                                    resolveAns,
-                                    stringDecoder,
-                                ) as T;
-                            } else {
-                                resolveAns = valueFromSplitPointer(
-                                    resolveAns.high!,
-                                    resolveAns.low!,
-                                    stringDecoder,
-                                ) as T;
-                            }
-                        }
-
+                        resolveAns = this.getValueToResolve(
```

Comment:
Is this refactoring related to `scan` command API?

Reply:
yes, this func is called again https://github.com/avifenesh/valkey-glide/blob/d59c92dec5202baf3bc692d23a60e28906874dcb/node/src/GlideClusterClient.ts#L371